libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java

```diff
@@ -390,6 +390,9 @@ public Bitmap getProfilePhoto() {
      * Fetches this user's profile photo from the server and stores it in the cache.
      */
     public void downloadProfilePhoto() {
```

Comment:
That code came in 6.2, could it throw exception for apps on 6.2?

Reply:
Nope. `photoUrl` can never really be `null`. Our dummy test account doesn't have that field populated locally. I could add that field in the dummy account.

Reply:
Agreed, the dummy test account should be more realistic.

Reply:
Will do.